regression/goto-analyzer/variable-sensitivity-asserts-02/test-value-sets.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+main.c
+--variable-sensitivity --vsd-values set-of-constants --verify
+^EXIT=0$
+^SIGNAL=0$
+\[main.assertion.1\] line 13 x == y: UNKNOWN
```

Comment:
These I agree with because both sets will be `{1, 3}` and you can't determine the equality or inequality of these.

Reply:
Again this is existing behaviour. The UNKNOWN arises because we try all the combinations of the values in the sets, in this case, `1 == 1` `1 == 3` `3 == 1` `3 == 3`, and because we have both true and false expressions, the result is unknown.